specs/merge/beacon-chain.md

```diff
@@ -289,6 +296,7 @@ def initialize_beacon_state_from_eth1(eth1_block_hash: Bytes32,
         timestamp=eth1_timestamp,
         receipt_root=Bytes32(),
         logs_bloom=ByteVector[BYTES_PER_LOGS_BLOOM](),
+        randao=eth1_block_hash,
```

Comment:
We could define `randao_seed = eth1_block_hash`, and use it in both the `randao_mixes` initialization and the `randao` field here. To clarify it's being uses as a seed, and not a copy-paste error.